tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py25,py26,py27,pypy,py32,py33,py34,lint
```

Comment:
Or ideally, we would fine a base Docker image (e.g. older version of Ubuntu) where we can easily also install Python 2.5 or Python 2.5 PPA which we can use with a newer version of Ubuntu,